rentals.rb

```diff
@@ -0,0 +1,16 @@
+require './book'
+require './person'
+
+# rentals class
+class Rental
+  attr_accessor :date
+
+  def initialize(date)
+    @date = date
+    @person = Person.new
+    @book = Book.new
+  end
+
+  belongs_to :book
+  belongs_to :person
+end
```

Comment:
- [ ] <b>REQUIRED:</b> There is no `many-to-many` relationship between `Person` and `Book` using the intermediate class `Rental`.
   - `belongs_to ` keyword you are using on line 14-15  is not supported. that's because it is only supported in Rails. to create `belongs_to ` relationship Here, you have to add the following code in The `Rental` class.
        ```ruby
            attr_accessor :date, :book, :person
            
           # The constructor of Rental should be modified so Book and Person are set in it to break 
           # the many-to-many relationship
            def initialize(title, author)        
                @book = book
               # The rental is adding itself to the @book achieving the 'belongs-to' association
                book.rentals << self
                            
               @person = person
               # The rental is adding itself to the @person the 'belongs-to' association
               person.rentals << self
            end
       ```